mne/viz/epochs.py

```diff
@@ -177,14 +177,14 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
             if colorbar:
                 ax3 = plt.subplot2grid((3, 10), (0, 9), colspan=1, rowspan=3)
         if scale_vmin:
-            vmin *= scalings[ch_type]
+            this_vmin = vmin * scalings[ch_type]
         if scale_vmax:
-            vmax *= scalings[ch_type]
+            this_vmax = vmax * scalings[ch_type]
         im = ax1.imshow(this_data,
                         extent=[1e3 * epochs.times[0], 1e3 * epochs.times[-1],
                                 0, len(data)],
                         aspect='auto', origin='lower', interpolation='nearest',
-                        vmin=vmin, vmax=vmax, cmap=cmap)
+                        vmin=this_vmin, vmax=this_vmax, cmap=cmap)
```

Comment:
You're right. There's also the logical problem of dealing with ``vmin`` and ``vmax`` when different kinds of channels are plotted at once (grads and mags for example). I guess we'll just have to leave it for the user to sort out.
@mmagnuski are you ok for closing this in favor of https://github.com/mne-tools/mne-python/pull/3290 (see https://github.com/mne-tools/mne-python/pull/3290/files#diff-19d699ab3f2a1990a1b4aa1928b6e66aR187)? I'll just add a test for the error pointed out by @jona-sassenhagen there